.github/actions/install_requirements/action.yml

```diff
@@ -10,6 +10,9 @@ inputs:
   python_version:
     description: Python version to install
     default: "3.x"
+  artifact_name:
+    description: A user friendly name to give the produced artifacts
+    default: "tests"
```

Comment:
Might be better off without a default here :shrug:

Reply:
Yh, I was only being lazy, I'll remove the default